inc/functions/content/profile-details-page-content.php

```diff
@@ -0,0 +1,80 @@
+<?php
+/**
+ * User: mdiaconita
+ * ------------------ CONTENTS ---------------------------
+ * 1. Main user content area
+ * 2. Generate extra textareas
+ */
+function profile_single_page_content($position, $contact)
+{
+    global $post;
+    if (function_exists('get_post_meta')
+        && function_exists('get_the_category')
+        && function_exists('the_content')
+        && function_exists('the_post_thumbnail')
+        && function_exists('the_title')
+        && function_exists('post_class')
+        && function_exists('has_post_thumbnail')
+        && function_exists('the_ID')
+    ) {
+
+        ${$position} = get_post_meta($post->ID, $position, true);
+        ${$contact} = get_post_meta($post->ID, $contact, true);
+
+        if (isset(${$position}) && !empty(${$position}) || isset(${$contact}) && !empty(${$contact})) { ?>
+            <article id="post-<?php the_ID(); ?>" <?php post_class(); ?>>
+                <div class="entry-header">
+                    <h1><?php the_title(); ?></h1>
+                </div>
+                <div class="entry-content clearfix">
+                    <div class="mobile-dev col-xs-6 col-sm-4 col-md-4 col-lg-4">
+                        <?= profile_feature_image('/../../img/profile-fall-back.jpg', has_post_thumbnail()); ?>
```

Comment:
Try and be consistent with using the php echo shortcut. It has been used in some places and not others.